resources/graphClient/graphClient.css

```diff
@@ -12,12 +12,26 @@
 html {
```

Comment:
The colors in the text box seem backwards. For example this is what it looks like in dark theme:
![screen shot 2017-12-08 at 3 50 37 pm](https://user-images.githubusercontent.com/11282622/33789773-f698d67c-dc2f-11e7-9c18-68325860222c.png)

But This is what the colors look like when I'm editing a normal file (Notice the _text_ is the light color and the background is dark):
![screen shot 2017-12-08 at 3 55 24 pm](https://user-images.githubusercontent.com/11282622/33789843-4e57fb7c-dc30-11e7-9a85-e8a6593f0228.png)

Reply:
It's intentional based on the original design from UX. Agreed it doesn't look as good with this theme.  I'll be discussing stuff like that next week and can adjust.